pkg/virt-launcher/virtwrap/converter/converter.go

```diff
@@ -283,6 +282,20 @@ func setReservation(disk *api.Disk) {
 	}
 }
 
+func setErrorPolicy(diskDevice *v1.Disk, disk *api.Disk) error {
+	if diskDevice.ErrorPolicy == "" {
+		log.Log.Infof("Setting error policy to stop for disk %s", diskDevice.Name)
+		disk.Driver.ErrorPolicy = v1.ErrorPolicyStop
+		return nil
+	}
+	if v1.ErrorPolicyStop != v1.ErrorPolicyStop && v1.ErrorPolicyStop != v1.ErrorPolicyIgnore &&
```

Comment:
`diskDevice.ErrorPolicy`? Also needs a test.

supernit - Would this more readable as a switch?

```go
switch diskDevice.ErrorPolicy {
case v1.ErrorPolicyStop, v1.ErrorPolicyIgnore, v1.ErrorPolicyReport, v1.ErrorPolicyEnospace:
    disk.Driver.ErrorPolicy = diskDevice.ErrorPolicy
default:
    return fmt.Errorf("error policy %s not recognized", diskDevice.ErrorPolicy)
return nil
```